pkg/hierarchy/syspath_test.go

```diff
@@ -0,0 +1,62 @@
+package hierarchy
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_getParent(t *testing.T) {
+	type fields struct {
+		DeviceName string
+		SysPath    string
+	}
+	tests := map[string]struct {
+		fields fields
+		want   string
```

Comment:
n_ : want can be parentDevice?  and wankOk can be hasParentDevice? for more clarity?

Reply:
Done.